src/test/resources/com/puppycrawl/tools/checkstyle/treewalker/InputTreeWalkerSuppressionCommentFilter.java

```diff
@@ -1,11 +1,19 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck
+
+
+com.puppycrawl.tools.checkstyle.filters.SuppressionCommentFilter
+checkCPP = false
+
+*/
 package com.puppycrawl.tools.checkstyle.treewalker;
 
 public class InputTreeWalkerSuppressionCommentFilter {
-    private int I;
+    private int I;    //violation
```

Comment:
Please add message part to violation message, as we do in other Inputs.
Please do this for all lines you updated in this PR

Reply:
@romani done